sites/fast-website/src/app/components/card-section/card-section.styles.ts

```diff
@@ -15,13 +15,17 @@ export const CardSectionStyles = css`
     ${display("grid")} :host {
         grid-template-rows: repeat(4, min-content);
         grid-auto-flow: var(--flow);
+        justify-content: center;
     }
 
     :host(:hover) ::slotted(site-feature-card) {
-        filter: blur(1px);
+        filter: saturate(0);
     }
 
     :host ::slotted(site-feature-card:hover) {
-        filter: blur(0);
+        cursor: pointer;
+        color: currentColor;
+        background-color: rgba(27, 27, 27, 1);
```

Comment:
should we be using a design-system color here?

Reply:
I couldn't find a design system option that matched the black in the Figma design, but I am happy to change it if one can be recommended.  The closest I found was more of a charcoal grey which had more of a washed out look than a look of focus.

Reply:
yeah I am on the same page with you. I am just unsure if the priority is using a color in the design system or a color that is more closely aligned with the figma.

Reply:
Let's definitely follow up on that to find out. Even if it's a unique color not in the core design system, I think it would make sense to extract this into a CSS variable.

Reply:
The question I tend to ask in these situations are, “what are the implications of ____?” In this case, if we set a static color, it will not theme with the rest of the page. That can create accessibility issues depending on how the page changes, etc. Thats just one that can be identified here without digging in, but I think it’s an important one. We should leverage the system here to ensure appropriate contrast. If we need a new design system provider to do so, we can do that. My guess here is that this is likely just a mistake or there is possibly a lack of awareness. When it comes to colors on the site, let’s be sure to leverage the system rather than hardcode colors.

Reply:
<img width="619" alt="Screen Shot 2020-05-26 at 8 40 21 AM" src="https://user-images.githubusercontent.com/26206206/82920956-ce7e9100-9f2c-11ea-9277-4946f32b4b1c.png">
neutral-fill-hover

<img width="619" alt="Screen Shot 2020-05-26 at 8 41 17 AM" src="https://user-images.githubusercontent.com/26206206/82921012-d5a59f00-9f2c-11ea-92b7-943cbc9f981c.png">
rgba

I initially tried neutral-fill-hover which is the first image, but that does look notably different than the design.  Since the design system is still being implemented I opted to match the design in the interim.  I've changed the rgba to neutral-fill-hover.  If there is a design system variable better suited here, please let me know what that may be.